### ISSUE TITLE
Hashing of passwords in database

### ISSUE THREAD

**Issue by fkoolhoven** [maresverbrugge/Transcendence#28]
Any passwords stored in the database must be hashed. We must use a strong hashing algorithm.

**Comment by maresverbrugge** [comment]
in the subject it says ⚠️  "Please make sure you use a strong password hashing algorithm" ⚠️

**Comment by fkoolhoven** [comment]
We are using bcrypt https://en.wikipedia.org/wiki/Bcrypt

**Comment by fhuisman** [comment]
Yes i think it's a strong algorithm! And its implemented and working :)
